transport_nantes/authentication/tests.py

```diff
@@ -1,3 +1,163 @@
-from django.test import TestCase
+from django.test import TestCase, RequestFactory
+
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.contrib.sites.models import Site
+import authentication.views as views
 
 # Create your tests here.
+
+# TODO Test good printing of pages (first print, post, redirect) and also page needing token from mail
+# TODO Test adding of new user if account doesn't exist and if account already exists
+class UserTest(TestCase):
+
+    # Add new user and test site page shown
+    def test_add_new_user(self):
+
+        # Set site into Site table
+        site = Site.objects.get(id=1)
```

Comment:
Where was this set?

Reply:
This changes the value "exemple.com" to "localhost:8000" in Site model. If not done get_current_site in send_activation function fails and all the test fails also